lib/email/utils/helpers.js

```diff
@@ -39,13 +70,34 @@ function logFlowEventFromMessage(log, message, type) {
       flow_time: elapsedTime
     }
 
+    // If a locale exists and log it
```

Comment:
nit: "Check if a locale exists.."?  Although TBH I'm not sure we need the comment there at all.